src/main/java/frc/robot/autochooser/chooser/Nt4AutoValidationChooser.java

```diff
@@ -24,20 +24,20 @@ private void setupListeners() {
         Nt4AutoEventProvider provider = (Nt4AutoEventProvider) getProvider();
         SmartShuffleboard.put(Nt4AutoEventProvider.AUTO_TAB_NAME, AUTO_ACTION_FEEDBACK_NAME, provider.getDefaultActionOption().toString())
                 .withPosition(2,2).withSize(2,1);
-        SmartShuffleboard.put(Nt4AutoEventProvider.AUTO_TAB_NAME, AUTO_LOCATION_FEEDBACK_NAME, provider.getDefaultLocationOption().toString())
+        SmartShuffleboard.put(Nt4AutoEventProvider.AUTO_TAB_NAME, AUTO_LOCATION_FEEDBACK_NAME, provider.getDefaultLocationOption().getShuffleboardName())
```

Comment:
Why not use `getShuffleboardName` on the other commands as well?

Reply:
field location depends on team color, Auto action does not